addons/l10n_pe/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright 2019 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from . import res_partner
+from . import res_district
```

Comment:
name properly the file as the model please.

Reply:
It will be "res_city_district" or "l10n_pe_res_city_district"?

Reply:
Done